sdks/java/io/hbase/pom.xml

```diff
@@ -52,29 +50,6 @@
               </goals>
               <configuration>
                 <createDependencyReducedPom>false</createDependencyReducedPom>
```

Comment:
This is a curious 'hack', we have this parent shade configuration that shades guava that produces a weird problem once you apply it to this module, and it stucks trying to produce the reduced pom for the HBase module. It keeps writing and deleting the reduced pom file. This configuration line is just to get rid of the issue. Curious thing is it seems to be a closed issue on the maven plugin but it is exactly what happens here.
https://issues.apache.org/jira/browse/MSHADE-148